src/main/java/com/icthh/xm/ms/otp/config/WebConfigurer.java

```diff
@@ -78,9 +81,7 @@ public void customize(WebServerFactory server) {
          * See the JHipsterProperties class and your application-*.yml configuration files
```

Comment:
fix this java doc (ServerProperties instead of JHipsterProperties)

Reply:
done